packages/bpk-component-text/src/BpkText.scss

```diff
@@ -0,0 +1,27 @@
+@import '~bpk-mixins';
+
+.bpk-text {
```

Comment:
So technically this isn't strictly bem right ...the only classes that are exposed here are modifier classes, i.e. the html will look like:
```
<span class="bpk-text--sm" />
```
Whereas strict BEM would be:
```
<span class="bpk-text bpk-text--sm" />
```

This comment is here more as an observation as opposed to a requests to change ... thoughts?